### PR TITLE
app-misc/ca-certificates: try to unbreak this for libressl

### LINKED ISSUE #61: libressl ebuild should run/package c_rehash
New installs building on libressl are going to have things failing all over the place (wget and curl to name a few) as this ebuild doesn't properly run c_rehash to prep the cert directory (nor does libressl include the script).

My friend hit the problem a couple nights ago on his rpi, lots of troubleshooting to see it was this.  Filing this for him.

### PR DESCRIPTION
**Pull request by hasufell**
This could/should fix #61.

### CODE REVIEW

**Comment by hasufell** [comment]
@blueness @hannob comments?

**Comment by hasufell** [comment]
advantages:
* doesn't use debian hackery (c_rehash and update-ca-certificates scripts are completely unnecessary now... instead we just need a single sed-call in src_install)
* uses the expected standard location for the bundle pem file
* doesn't even need the hash-style ca-directory, because the bundle pem file is in the correct location now
* doesn't need any pkg_postinst stuff
* doesn't need to deal with broken symlinks
* gives more control to the user by introducing the trusted-db USE flag and by omitting the hash-style ca-directory creation

disadvantages:
* this is a forked ebuild
* some ebuilds like curl hardcode the bundle pem file location and as such we need a symlink workaround which sucks

also see
https://www.happyassassin.net/2015/01/12/a-note-about-ssltls-trusted-certificate-stores-and-platforms/

**Comment by mgorny** [comment]
You submit that to Gentoo, you hasufell!

File: app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		newins "${MY_D}"/SPI_Inc..crt spi-cacert-2008.crt
+	fi
+
+	doman ca-certificates/sbin/*.8
+	dodoc ca-certificates/debian/README.* \
+		ca-certificates/examples/ca-certificates-local/README
+
+	if use trusted-db ; then
+		find "${ED%/}"/var/lib/ca-certificates/ -type f \
+			-exec sed -e '$a\' '{}' >> "${ED%/}"/etc/ssl/cert.pem + \
```
Comment (mgorny) [inline app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild]: Isn't that invalid syntax?

File: app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+MY_D=${S}/${PN}/mozilla
+
+src_unpack() {
+	default
+	mv ${PN}-*/ ${PN} || die
+}
+
+src_prepare() {
+	if use cacert ; then
+		pushd "${S}"/nss-${NSS_VER} >/dev/null
```
Comment (mgorny) [inline app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild]: `||die`.

File: app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild (hunk 1 above)
Comment (hasufell) [inline app-misc/ca-certificates/ca-certificates-20140927.3.17.2-r1.ebuild]: why?

**Pull request merged**